lib/version.rb

```diff
@@ -1,3 +1,4 @@
 module DangerSwiftlint
-  VERSION = "0.3.0".freeze
+  VERSION = "0.16.1".freeze
```

Comment:
And of course, versions prior to `0.16.1` wouldn't benefit from this behavior and if you have `gem "0.3.0"` on your Gemfile that will still work as it is today and will download the latest `swiftlint` if it is not installed yet.